wordpress-mobile/WordPress-iOS
# Merge 15.2 release notes and beta 2 into develop

Merges final release notes for 15.2 into `develop` as well as a beta fix previously approved in #14407.

**To Test:**
- Ensure there are new release notes
- Ensure that the changed version numbers go up
